rns_shell/compositor/Compositor.h

```diff
@@ -61,7 +61,7 @@ class Compositor {
     SharedLayer rootLayer_;
     std::unique_ptr<WindowContext> windowContext_;
     sk_sp<SkSurface> backBuffer_;
-    uint64_t nativeWindowHandle_;
+    GLNativeWindowType nativeWindowHandle_;
```

Comment:
in createWindowContext() we may not require the typecast on nativeWindowHandle_ now ?

Reply:
Done